Week02/lesson04/homework/Layout Challenge/Two Column/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+
+<link rel="stylesheet" href="styles.css">
+
+<head>
+
+  <title>
+  Two Column
+  </title>
+
+</head>
+
+<body>
+
+  <div class="top">
+  </div>
+
+  <div class="middle top">
```

Comment:
Loving the application of multiple classes, but given that `.top` ultimately styles your lovely pink squares, I think maybe a name like `.box` or `.block` would've been better. When I read your code, your class names should give me an idea of what sort of styling they do before I even look at your CSS